spec/features/social_networking_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+feature 'Social Networking' do
+  context "with authenticated user" do
+    let(:user) { create(:user) }
+    background do
+      valid_sign_in(user)
+      @other_user = create(:user)
+      @futurama = create(:video, title: 'futurama')
```

Comment:
I would not use instance variables in specs at all. They carry and may influence other specs. For exactly this reason - to have other specs isolated completely, use `let` or `given` in feature specs. or `let!` or `given!`.